java-lib/src/main/java/org/clink/util/ClinkJNA.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2021 The Clink Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.clink.util;
```

Comment:
OK. I'll make the change.

Do you think we should further place these files in sub packages like `org.clink.jna.linalg`? So far as I can see we may have classes like `DenseVector` or `Matrix` to be placed in this package in future, and it might be improper to mix them with `ClinkJNA` in the same package.

Reply:
For now I would slightly prefer to start simple and just put these files in the `org.clink.jna`. In the future we can add more sub-directory as the number of files grow larger.